src/main/java/io/jenkins/plugins/gitlabbranchsource/GitLabSCMNavigator.java

```diff
@@ -337,24 +335,12 @@ public static FormValidation doCheckProjectOwner(@QueryParameter String projectO
             GitLabApi gitLabApi = null;
             try {
                 gitLabApi = GitLabHelper.apiBuilder(serverName);
-                User user = gitLabApi.getUserApi().getUser(projectOwner);
-                if(user == null) {
-                    throw new GitLabApiException("Invalid User");
-                }
-                return FormValidation.ok(projectOwner + " is a valid user");
+                GitLabOwner gitLabOwner = GitLabOwner.fetchOwner(gitLabApi, projectOwner);
+                return FormValidation.ok(projectOwner + " is a valid " + gitLabOwner.getWord());
             } catch (NoSuchFieldException e) {
                 e.printStackTrace();
-            } catch (GitLabApiException e) {
-                try {
-                    Group group = gitLabApi.getGroupApi().getGroup(projectOwner);
-                    String groupNamespace = group.getFullName();
-                    if(groupNamespace.indexOf('/') == -1) {
-                        return FormValidation.ok(groupNamespace + " is a valid group");
-                    }
-                    return FormValidation.ok(groupNamespace + " is a valid subgroup");
```

Comment:
If we want to preserve this could properly be included in GitlabGroup's word implementation :sweat_smile: